### ISSUE TITLE
PropertyFieldOfficeVideoPicker wrong video portal url

### ISSUE THREAD

**Issue by maxmurash** [OlivierCC/sp-client-custom-fields#11]
If webpart with PropertyFieldOfficeVideoPicker is added not at the tenant root site collection, the picker can't open a video dialog.

The problem is here: https://github.com/OlivierCC/sp-client-custom-fields/blob/master/src/PropertyFieldOfficeVideoPickerHost.tsx#L246

```
var iframeUrl = this.props.context.pageContext.web.absoluteUrl;
iframeUrl += '/portals/hub/_layouts/15/VideoAssetDialog.aspx?list=&IsDlg=1';
```
Video portal url is: **{RootTenantSite}/portals/hub/**, not **{someWeb}/portals/hub/**

I suppose a good option is to use "Video REST API" and the method `{RootSite}/_api/VideoService.Discover`, it returns "VideoPortalUrl" and "IsVideoPortalEnabled" params.

In addition, it would be great to add a configurable parameter with the video portal url to IPropertyFieldOfficeVideoPickerProps.

Thanks a lot.
Max.

**Comment by OlivierCC** [comment]
You are totally right Max, thanks a lot for this report. I will integrate these modifications in the next release.
Thanks
Olivier
